netmanager/src/views/pages/Preferences/Preferences.js

```
@@ -0,0 +1,579 @@
+    fetchSelectedSites();
+    if (allSites.length === 0 && activeNetwork) {
+      dispatch(loadSitesSummary(activeNetwork.net_name));
+    }
+  }, [dispatch, activeNetwork]);
+
+  const fetchSelectedSites = async () => {
+    setIsLoading(true);
+    setError(null);
+    try {
+      const response = await getDefaultSelectedSitesApi();
+      if (response && response.selected_sites) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)